### DISCUSSION TITLE
Restore Json files backups

### DISCUSSION THREAD

**Discussion by ilbarone87** [requarks/wiki#5870] [Help / Questions] [answered]
Hello all
I backed up all my wikijs pages, settings. users, navigation, etc... 
These are in tarball or json format.
How do I import them in the new instance? 
Thanks

**Comment by NGPixel** [comment, marked as answer]
You can't. The tarball is meant for import in the future v3 and for general secondary backup in a readable format.

To migrate an instance to a new one, you need to backup and import the database (see https://docs.requarks.io/install/transfer)
